src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -57,6 +57,22 @@
  * <pre>
  * &lt;module name="NonEmptyAtclauseDescription"/&gt;
  * </pre>
+ * <pre>
+ * class Test
+ * {
+ * &#47;**
+ * *
+ * * &#64;param a Some javadoc // OK
+ * * &#64;param b // Violation - At-clause should have a non-empty description.
+ * * &#64;deprecated // Violation - At-clause should have a non-empty description.
+ * * &#64;throws Exception // Violation - At-clause should have a non-empty description.
+ * *&#47;
+ * public int method(String a, int b) throws Exception
+ * {
+ * return 1;
```

Comment:
please make proper indentation for code in javadoc.

please run javadoc jdk tool over this file and share result (as image) in PR description. javadoc should render properly and code is easy to read there.